src/go/cmd/http-relay-client/client/client.go

```diff
@@ -643,8 +655,11 @@ func (c *Client) handleRequest(remote *http.Client, local *http.Client, pbreq *p
 				log.Printf("[%s] Failed to post response to relay: %v", *resp.Id, err)
 			},
 		)
-		if _, ok := err.(*backoff.PermanentError); ok {
+		if _, ok := err.(*RelayServerError); ok {
 			// A permanent error suggests the request should be aborted.
+			log.Printf("[%s] %s", *resp.Id, err)
+			log.Printf("[%s] Closing backend connection", *resp.Id)
```

Comment:
Please only log a single line:
log.Printf("[%s] Relay error: %v, closing backend connection", *resp.Id, err)

Reply:
updated in [301b04d](https://github.com/googlecloudrobotics/core/pull/222/commits/301b04d5a0c3449018d5207a18f145e812cf4d02)